loader/loader.go

```diff
@@ -89,7 +90,7 @@ type Package struct {
 // Load loads the given package with all dependencies (including the runtime
 // package). Call .Parse() afterwards to parse all Go files (including CGo
 // processing, if necessary).
-func Load(config *compileopts.Config, inputPkgs []string, clangHeaders string, typeChecker types.Config) (*Program, error) {
+func Load(config *compileopts.Config, inputPkg string, clangHeaders string, typeChecker types.Config) (*Program, error) {
```

Comment:
I turned this into a single package for a few reasons:
1. It is used for single package only everywhere.
2. It is easier to deal with a single package.
3. [This error](https://github.com/tinygo-org/tinygo/pull/2885/files#diff-8e9af1e483632eac17610237caaa5f879ea7bfb3e1e1ea228e45a17c770b9432L231) is only looking at one package and it is actually thought of a single package